### PR TITLE
Login page tweak

### PR DESCRIPTION
**Pull request by PromoFaux**
**By submitting this pull request, I confirm the following:** 

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
**What does this PR aim to accomplish?:**
Simple tweak to make thigns less confusing for some.

Before PR:

image

After PR:
image

fixes https://github.com/pi-hole/AdminLTE/issues/1572

### CODE REVIEW

**Comment by PromoFaux** [comment]
Oops, forgot to add mobile screenshots.

Before:
image


After:
image

File: scripts/pi-hole/php/loginpage.php [hunk 1]
```diff
@@ -30,34 +30,36 @@
...
+                <label for="logincookie">Remember me for 7 days</label>
+              </div>
           </div>
         </div>
         <br>
         <div class="row">
           <div class="col-xs-12">
             <div class="box box-<?php if (!$wrongpassword) { ?>info<?php } else { ?>danger<?php }
-            if (!$wrongpassword) { ?> collapsed-box<?php } ?> box-solid">
+                                                                                        if (!$wrongpassword) { ?> collapsed-box<?php } ?>">
```
Comment (DL6ER) [inline scripts/pi-hole/php/loginpage.php]: Many new spaces?

Also, this should be combined with the `if()` directly before this one.

File: scripts/pi-hole/php/loginpage.php [hunk 2]
```diff
@@ -30,34 +30,36 @@
           <input type="password" id="loginpw" name="pw" class="form-control" placeholder="Password" autofocus>
           <span class="fa fa-key form-control-feedback"></span>
         </div>
+        <div class="row">
+          <div class="col-xs-12 col-md-12">
+            <button type="submit" class="btn btn-primary form-control"><i class="fas fa-sign-in-alt"></i>&nbsp;&nbsp;&nbsp;Log in</button>
+          </div>
+        </div>
+        <br>
```
Comment (DL6ER) [inline scripts/pi-hole/php/loginpage.php:38]: Why is there a `<br>` between the rows? To get some extra spacing? Maybe we should use some `margin-bottom` on the upper element (the `row` div). I think this is more how it should be done.

File: scripts/pi-hole/php/loginpage.php [hunk 3]
```diff
@@ -73,4 +75,4 @@ class="fa <?php if ($wrongpassword) { ?>fa-minus<?php } else { ?>fa-plus<?php }
       </form>
     </div>
   </div>
-</div>
+</div>
```
Comment (DL6ER) [inline scripts/pi-hole/php/loginpage.php]: Newline at EOF was removed accidentally.

File: scripts/pi-hole/php/loginpage.php (hunk 1 above)
Comment (PromoFaux) [reply scripts/pi-hole/php/loginpage.php]: I am not sure how I managed the many extra spaces!!

File: scripts/pi-hole/php/loginpage.php (hunk 2 above)
Comment (PromoFaux) [reply scripts/pi-hole/php/loginpage.php:38]: I just use the same as the existing `<br>` here:

https://github.com/pi-hole/AdminLTE/pull/1583/files#diff-22686796f3faafaceb91a6ff8a199efbL50

**Review by DL6ER — APPROVED** [review]

**Pull request merged**